src/test/scala-2.11/org/apache/james/gatling/smtp/scenari/common/Configuration.scala

```diff
@@ -0,0 +1,10 @@
+package org.apache.james.gatling.smtp.scenari.common
+
+import org.apache.james.gatling.smtp.{SmtpActionBuilder, SmtpProtocolBuilder}
+
+object Configuration {
+
+  val smtp = SmtpProtocolBuilder.default.build()
+
+  def smtp(requestName: String) = new SmtpActionBuilder(requestName, null, null)
```

Comment:
I don't think it's related to a scenario, it's part of smtp protocol implementation, isn't it ?

Reply:
Because the SMTP server you are willing to test may not have a standard configuration (ssl, other port, ...), so it really depend on the execution of the test.
It's seems legit to have such configuration in the `test` package.